### DISCUSSION TITLE
Is there easy way to get the actual size of a node?

### DISCUSSION THREAD

**Discussion by geohuz** [xyflow/xyflow#3097] [General]
I want to create subgraph dynamically based on current selected node, the group node size should be determined dynamically, but it looks like no way to get the actual size from the doc.

**Comment by moklick** [comment]
you can access with and height from a node itself:

```js
const { width, height } = node;
```

**Reply by duke-arioch** [reply]
I have dynamic nodes I am pulling in from  REST service. I have a custom Node type, which draws a fancy representation of the incoming data. If I load the data with a zero width/height it does not display - so I set the width/height to 100/100 on load. I don't see any way to get the actual rendered width/height - which I think is what the original poster is going for?
